sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/AbstractDistributeTest.java

```diff
@@ -211,16 +211,16 @@ public String answer(InvocationOnMock invocation) throws Throwable {
 
             @Override
             public Void answer(InvocationOnMock invocation) throws Throwable {
-            when(mDialog.isShowing()).thenReturn(true);
-            return null;
+                when(mDialog.isShowing()).thenReturn(true);
```

Comment:
is that tabs? we use only spaces to indent, not sure if github is tricking me.

Reply:
No, it's spaces. GitHub diffs seem confusing a little bit.

Please see the screenshot:
![Screenshot](https://user-images.githubusercontent.com/33692429/35828816-f0866710-0ad1-11e8-85b5-d10649bec6b4.png)